application/database/migrations/20201230231550_add_controle_cobrancas.php

```diff
@@ -0,0 +1,107 @@
+            'idCobranca' => [
+                'type' => 'INT',
+                'constraint' => 11,
+                'null' => false,
+                'auto_increment' => true
+            ],
+            'charge_id' => [
+                'type' => 'INT',
+                'constraint' => 11,
+                'null' => false,
+            ],
+            'conditional_discount_date' => [
+                'type' => 'DATE',
+                'null' => false,
+            ],
+            'created_at' => [
+                'type' => 'DATETIME',
+                'null' => false,
+            ],
+            'custom_id' => [
+                'type' => 'INT',
+                'constraint' => 4,
+                'null' => true,
+            ],
+            'expire_at' => [
+                'type' => 'DATE',
+                'null' => false,
+            ],
+            'message' => [
+                'type' => 'VARCHAR',
+                'constraint' => 255,
+                'null' => true,
+            ],
+            'payment_method' => [
+                'type' => 'VARCHAR',
+                'constraint' => 36,
+                'null' => true,
+            ],
+            'payment_url' => [
+                'type' => 'VARCHAR',
+                'constraint' => 255,
+                'null' => true,
+            ],
+            'request_delivery_address' => [
+                'type' => 'VARCHAR',
+                'constraint' => 64,
+                'null' => true,
+            ],
+            'status' => [
+                'type' => 'VARCHAR',
+                'constraint' => 36,
+                'null' => true,
+            ],
+            'total' => [
+                'type' => 'VARCHAR',
+                'constraint' => 15,
+                'null' => true,
+            ],
+            'barcode' => [
+                'type' => 'VARCHAR',
+                'constraint' => 255,
+                'null' => true,
+            ],
+            'link' => [
+                'type' => 'VARCHAR',
+                'constraint' => 255,
+                'null' => true,
+            ],
+            'payment' => [
+                'type' => 'VARCHAR',
+                'constraint' => 64,
+                'null' => true,
+            ],
+            'pdf' => [
+                'type' => 'VARCHAR',
+                'constraint' => 255,
+                'null' => true,
+            ],
+            'vendas_id' => [
+                'type' => 'INT',
+                'constraint' => 11,
+                'null' => true,
+            ],
+            'os_id' => [
+                'type' => 'INT',
+                'constraint' => 11,
+                'null' => true,
+            ],
+        ]);
+        $this->dbforge->add_key("idCobranca", true);
+        $this->dbforge->create_table("cobrancas", true);
+        $this->db->query('ALTER TABLE `cobrancas` ENGINE = InnoDB');
+    }
+
+    public function down()
+    {
+        $this->dbforge->drop_table('cobrancas');
+    }
+}
```

Comment:
Se não for muito incômodo ou difícil, eu acho que talvez seria melhor deixar as colunas em portugues por causa que nas demais tabelas são em portugues, só para manter o padrão.